### PR TITLE
Cex from datalake test

### PR DESCRIPTION
**Pull request by tonagiz**
Merging labels from datalake-test.cex_labels_20241116

### CODE REVIEW

File: assets/cex/mobile.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "address": "EQDOPNz5UIm3XuXOmCSk_1BvQLdQkMS-lmN8K404sQlyJPrd",
+        "label": "mobile",
+        "organization": "mobile",
+        "category": "CEX",
```
Comment (shuva10v) [inline assets/cex/mobile.json:6]: merchant

File: assets/merchants/bc.game.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "address": "EQA-B8bcDEegRwp_cIrrWrFjHxCGScn69h7XWPgkoXGSIj3Q",
+        "label": "bc.game",
+        "organization": "bc.game",
+        "category": "CEX",
```
Comment (shuva10v) [inline assets/merchants/bc.game.json:6]: merchant

File: assets/merchants/sportsbet.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "address": "EQAscR8-n759ngxVUOmPDOzue7CfBt_xIr-bi7kex_yd5Hnu",
+        "label": "sportsbet.io",
+        "organization": "sportsbet.io",
+        "category": "CEX",
```
Comment (shuva10v) [inline assets/merchants/sportsbet.json:6]: merchant?

**Review by shuva10v — CHANGES_REQUESTED** [review]

File: assets/cex/mobile.json (hunk 1 above)
Comment (tonagiz) [reply assets/cex/mobile.json]: changed to `merchant`

File: assets/merchants/bc.game.json (hunk 2 above)
Comment (tonagiz) [reply assets/merchants/bc.game.json]: changed to `merchant`

File: assets/merchants/sportsbet.json (hunk 3 above)
Comment (tonagiz) [reply assets/merchants/sportsbet.json]: changed to `merchant`

**Pull request merged**